===================================================================
RCS file: /cvs/openafs/src/bucoord/bc.p.h,v
retrieving revision 1.8
retrieving revision 1.9
diff -u -r1.8 -r1.9
--- openafs/src/bucoord/bc.p.h	2009/03/16 13:32:04	1.8
+++ openafs/src/bucoord/bc.p.h	2009/04/27 19:17:14	1.9
@@ -144,4 +144,4 @@
 #define	BC_MAXSIMDUMPS	    64
 #define BC_MAXPORTS	    128	/* max number of port offsets for volrestore */
 /* debugging support */
-#define	dprintf(x)
+#define	afs_dprintf(x)
===================================================================
RCS file: /cvs/openafs/src/bucoord/ubik_db_if.c,v
retrieving revision 1.17
retrieving revision 1.18
diff -u -r1.17 -r1.18
--- openafs/src/bucoord/ubik_db_if.c	2009/03/27 13:45:49	1.17
+++ openafs/src/bucoord/ubik_db_if.c	2009/04/27 19:17:14	1.18
@@ -574,7 +574,7 @@
 
     fileSize = (afs_int32) filesize(ctPtr->textStream);
 
-    dprintf(("filesize is %d\n", fileSize));
+    afs_dprintf(("filesize is %d\n", fileSize));
 
     rewind(ctPtr->textStream);
 
@@ -1350,7 +1350,7 @@
 	ERROR(errno);
 #endif
 
-    dprintf(("file is %s\n", tmpFileName));
+    afs_dprintf(("file is %s\n", tmpFileName));
 
   normal_exit:
     return code;
===================================================================
RCS file: /cvs/openafs/src/config/debug.h,v
retrieving revision 1.4
retrieving revision 1.5
diff -u -r1.4 -r1.5
--- openafs/src/config/debug.h	2003/07/15 23:14:55	1.4
+++ openafs/src/config/debug.h	2009/04/27 19:17:12	1.5
@@ -112,21 +112,21 @@
 
 #if defined(AFS_DEBUG)
 #if defined(lint)
-#define dprintf(flag, str) printf str
+#define afs_dprintf(flag, str) printf str
 #define dlprintf(flag, level, str) printf str
 #define dmprintf(flag, bit, str) printf str
 #else /* lint */
-#define dprintf(flag, str) \
+#define afs_dprintf(flag, str) \
       (void)((flag) ? \
 	     ( osi_dp str, osi_dp("\t%s, %d\n", __FILE__, __LINE__)):0)
-#define dlprintf(flag, level, str) dprintf(((flag) >= (level)), str)
-#define dmprintf(flag, bit, str) dprintf(((flag)&(1<<((bit)-1))), str)
+#define dlprintf(flag, level, str) afs_dprintf(((flag) >= (level)), str)
+#define dmprintf(flag, bit, str) afs_dprintf(((flag)&(1<<((bit)-1))), str)
 
 #endif /* lint */
 
 #else /* AFS_DEBUG */
 
-#define dprintf(flag, str)
+#define afs_dprintf(flag, str)
 #define dlprintf(flag, level,str)
 #define dmprintf(flag, bit, str)
 
===================================================================
RCS file: /cvs/openafs/src/sgistuff/rcmd.c,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -r1.1 -r1.2
--- openafs/src/sgistuff/rcmd.c	2005/08/16 17:58:27	1.1
+++ openafs/src/sgistuff/rcmd.c	2009/04/27 19:17:15	1.2
@@ -55,8 +55,8 @@
 #endif /* defined(AFS_HPUX_ENV) */
 #ifdef	TCP_DEBUG
 #include <sys/syslog.h>
-#	define	DPRINTF(args)	dprintf args
-dprintf(args)
+#	define	DPRINTF(args)	afs_dprintf args
+afs_dprintf(args)
      char *args;
 {
     char **argv;
